### PR TITLE
Issue3 user can edit entries - Omid Asadi 

### PR DESCRIPTION
**Pull request by omidzed**

### CODE REVIEW

**Comment by omidzed** [comment]
https://github.com/omidzed/code-journal/assets/136918229/ca8d3cdb-b09f-4bde-a031-687e676f399c

File: css/styles.css [hunk 1]
```diff
@@ -3,36 +3,48 @@ header {
...
 
 .app-name {
   align-items: left;
 }
 
 #entries-id {
-  font-size: 1.7rem;
-  padding: 1.5rem;
-  margin: 0 1rem;
+  font-size: 170%;
```
Comment (shawnkost) [inline css/styles.css:21]: You don't want to use percents for `font-size`. Either `px` or `rem` would be a better unit of measurement

File: css/styles.css [hunk 2]
```diff
@@ -3,36 +3,48 @@ header {
...
 
-h1 {
+.entries-list {
+  padding: 0;
+}
+
+#code-journal {
   color: white;
-  font-size: 2.1rem;
+  font-size: 230%;
```
Comment (shawnkost) [inline css/styles.css:32]: You don't want to use percents for `font-size`. Either `px` or `rem` would be a better unit of measurement

**Review by shawnkost — APPROVED** [review]
Approved, but left a couple comments!

**Pull request merged**